package.json

```diff
@@ -74,7 +74,7 @@
     "server": "node server.js",
     "lint": "gulp lint",
     "start": "gulp watch",
-    "test": "npm run lint && npm run build && intern-client config=tests/config/intern-node && intern-runner config=tests/config/intern-browser",
+    "test": "npm run lint && REDIS_INDEX=5 npm run build && REDIS_INDEX=5 intern-client config=tests/config/intern-node && REDIS_INDEX=5 intern-runner config=tests/config/intern-browser",
```

Comment:
Before this patch the build wasn't run with REDIS_INDEX=5. Should we still run it without the env variable?